### ISSUE TITLE
playing a video on file System freezes the UI

### ISSUE THREAD

**Issue by kcochibili** [rtoshiro/FullscreenVideoView#3]
I am trying to play a video stored on the file sytem, but when I attempt to play the video, the phone freezes, (until i remove the battery, and reinsert it)..

this is my code:
 `videoLayout.setVideoPath(savePath);'
`videoLayout.setShouldAutoplay(true);` (the problem still occurs even without autoPlay)

the video i am trying to play is 300kb in size
I am testing on a Samsung Galaxy S3, running Jelly bean

pls how can I fix this issue?

**Comment by rtoshiro** [comment]
Hi @kcochibili 
It is a little bit difficult without the code.
Can you test replacing your VideoLayout with the native android.widget.VideoView and see if it still happening? 
If native android.widget.VideoView works, can you test replacing it with FullscreenVideoView (instead of **Layout**) 
When the device freezes, are there any exception or anything in log?

**Comment by kcochibili** [comment]
When I  tested using  android.widget.`VideoView` the problem still occurred. So then i started thinking that there must be something wrong with my implementation, since the problem does not occur in the previous activity where I also played the same video.

The problem turned out to be that I did not call `VideoView.stopPlayback()` in the previous activity where I had a handle on this video file (The video was still playing when I launched a new activity that attempted to play the same video Using `FullscreenVideoLayout`).

So I was able to stop the freezing by first calling  `VideoView.stopPlayback()` before launching a new activity.

Thank you for the helpful diagnostic suggestion.that helped solved my problem.
